Source/ASEditableTextNode.mm

```diff
@@ -842,4 +842,24 @@ - (void)_delegateDidFinishEditing
     [_delegate editableTextNodeDidFinishEditing:self];
 }
 
```

Comment:
Can you please add a pragma to this line: `#pragma mark - UIAccessibilityContainer`